assets/web-ui/js/widget.js

```diff
@@ -453,6 +472,10 @@ class Widget {
         this.repaint();
     }
 
+    onHighlightedFunctionUpdate() {
+        this.repaint();
```

Comment:
It seems on click this scrolls to the top of the list again. Just wanted to note this.

Reply:
Also this prevents that I can copy the entry text via mouse (click + drag or doubleblick).

Reply:
@DanielRuf yes in the Flat profile view I confirm this annoying behavior.

Reply:
The "repaint" also causes the text to be unselected.

Reply:
Should I file an issue for that @NoiseByNorthwest?

Reply:
Yes, I'll also accept external contribution as long as the patch does not make things worst in terms of UX. It's unlikely that I'll fix this myself as I'm currently focusing on a full rewrite of the web UI which may be available within 6-12 months.